apps/web/lib/features/team-members-block-view.tsx

```diff
@@ -30,13 +30,13 @@ const TeamMembersBlockView: React.FC<Props> = ({
 			emptyMessage = t('common.NO_USERS_ONLINE');
 			break;
 		case 'running':
-			emptyMessage = 'No users are currently working.';
+			emptyMessage = t('common.NO_USERS_WORKING');
 			break;
 		case 'pause':
-			emptyMessage = 'No users are paused work at the moment.';
+			emptyMessage = t('common.NO_USERS_PAUSED_wORK');
```

Comment:
replace this `'common.NO_USERS_PAUSED_wORK'` with this `'common.NO_USERS_PAUSED_WORK'` in all files

Reply:
@Anishali2 , you repeated `'common.NO_USERS_PAUSED_wORK'`, can you give me more details?